PROJECT

```diff
@@ -1,5 +1,3 @@
-# SPDX-License-Identifier: MIT
```

Comment:
Let's add `PROJECT` to the list of generated files in the license-scanner-script:
https://github.com/mercedes-benz/garm-operator/blob/main/hack/verify-license.sh#L22

Reply:
fixed